src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -80,6 +80,11 @@ private string GetIncludePath(RuleSet parent)
             List<string> found = new List<string>();
             string expandedPath = PortableShim.Environment.ExpandEnvironmentVariables(_includePath);
 
+            if (PathUtilities.IsUnixLikePlatform)
+            {
+                expandedPath = expandedPath.Replace('\\', PortableShim.Path.DirectorySeparatorChar);
```

Comment:
Legacy ruleset files ported from Windows platform can contain Window style separator char (for example, our own Compiler project ruleset has this issue [here](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/CodeAnalysisRules.ruleset#L4)). We either need to require all users to manually fix their rulesets or do this fix in the ruleset parser.